hubble-fe/README.md

```diff
@@ -1,3 +1,3 @@
-# HugeGraph-Hubble
-
-Front-end implementation of HugeGraph
\ No newline at end of file
+1: add UI/index.tsx file and replace @baidu-ui to @ui
+2: GraphManagementList.tsx QueryAndAlgorithmLibrary.tsx 的 dropdown 重写
+3：Embedded 组件的 的修改
```

Comment:
we can create a new file CHANGELOG.md and move this content into it

Reply:
ok